config/checkstyle_non_main_files_checks.xml

```diff
@@ -38,4 +38,19 @@
       <property name="message"
                value="Two or more test cases are required for unit testing in Xpath "/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="&lt;!--[^ ]"/>
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required after comment start"/>
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="[^ ]--&gt;"/>
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required before comment end"/>
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value=" #[^ !]"/>
+        <property name="fileExtensions" value="sh, yml"/>
```

Comment:
I think yaml should be added, just in case

Reply:
done